### PR TITLE
El stone requested a pull 

### PR DESCRIPTION
**Pull request by thehapax**
@EL_Stone on telegram made a request for a pull for these two files to the latest dexbot release 0.7.25 however is not familiar with github, so I did it for him from a separate branch. Please connect him with regards to questions. I am just helping with the pull.

### CODE REVIEW

File: dexbot/basestrategy.py [hunk 1]
```diff
@@ -276,27 +255,33 @@ def calculate_center_price(self, center_price=None, asset_offset=False, spread=N
...
-            if balance < 0:
-                # With less of base asset center price should be offset downward
-                calculated_center_price = calculated_center_price / math.sqrt(1 + spread * (balance * -1))
-            elif balance > 0:
-                # With more of base asset center price will be offset upwards
-                calculated_center_price = calculated_center_price * math.sqrt(1 + spread * balance)
+            ticker = self.market.ticker()
+            highest_bid = ticker.get("highestBid")
+            lowest_ask = ticker.get("lowestAsk")
+            total = total_balance['quote'] * float(lowest_ask) + total_balance['base']
```
Comment (MarkoPaasila) [inline dexbot/basestrategy.py]: I would use cp rather than ask. No big difference though

**Comment by bitfag** [comment]
El Stone calculate_asset_offset() implementation adjusts more aggressively and it is not symmetrical. The current implementation is.

**Comment by bitfag** [comment]
I did a refactored version in #404

**Comment by joelvai** [comment]
This PR was included in #404

**Pull request closed without merging**